central/sensor/service/connection/connection_impl.go

```diff
@@ -177,6 +179,13 @@ func (c *sensorConnection) runRecv(ctx context.Context, grpcServer central.Senso
 			c.stopSig.SignalWithError(errors.Wrap(err, "recv error"))
 			return
 		}
+
+		if c.sensorEventHandler.getRateLimitMgr().limitClusterMsg(c.clusterID) {
+			errRateLimit := status.Error(codes.ResourceExhausted, "Stream message is rejected by sensor event rate limiter")
```

Comment:
Similar to the other one, we should bubble this up to Central's administrative events.